source/2020-04-10-the-ember-times-issue-143.md

```diff
@@ -46,14 +47,13 @@ Here at The Ember Times, we've been highlighting projects and teams who are usin
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [Check out new Ember CLI documentations 📚](https://cli.emberjs.com/release/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+In case you might not have known, the Ember.js website has [documentation dedicated to Ember CLI](https://cli.emberjs.com/release/). It covers both everyday and advanced uses that you will encounter when you write Ember apps or addons.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+In the past two weeks, [Mehul Kar (@mehulkar)](https://github.com/mehulkar) dedicated his time to [keep the documentation up to date](https://github.com/ember-learn/cli-guides/pulls?q=is%3Apr+is%3Aclosed+author%3Amehulkar+created%3A%3E%3D2020-03-15+) and introduce 2 new sections: how to [debug when CLI commands fail](https://cli.emberjs.com/release/advanced-use/debugging/) and how to [create custom CLI commands](https://cli.emberjs.com/release/writing-addons/custom-commands/).
```

Comment:
I think it'd be 
```suggestion
In the past two weeks, [Mehul Kar (@mehulkar)](https://github.com/mehulkar) dedicated his time to [keep the documentation up to date](https://github.com/ember-learn/cli-guides/pulls?q=is%3Apr+is%3Aclosed+author%3Amehulkar+created%3A%3E%3D2020-03-15+) and introduced 2 new sections: how to [debug when CLI commands fail](https://cli.emberjs.com/release/advanced-use/debugging/) and how to [create custom CLI commands](https://cli.emberjs.com/release/writing-addons/custom-commands/).
```

What do you think?

Reply:
Yep, thanks for catching it!